notebooks/docs.clj

```diff
@@ -0,0 +1,347 @@
+;; # 📓 Clerk Documentation
+(ns docs
+  {:nextjournal.clerk/toc true}
+  (:require [clojure.string :as str]
+            [next.jdbc :as jdbc]
+            [nextjournal.clerk :as clerk]
+            [nextjournal.clerk.parser :as parser]
+            [nextjournal.clerk.eval :as eval]
+            [nextjournal.clerk.analyzer :as ana]
+            [nextjournal.clerk.viewer :as v]
+            [weavejester.dependency :as dep]))
+
+;; ## ⚖️ Rationale
+
+;; ## 🚀 Getting Started
+
+;; ### ⏱ File Watcher
+
+;; ### 🔪 Editor Integration
+
+;; For even better flow states, we recommend you bind clerk/show! to a shortcut in your favorite editor:
+
+
+
+;; ## 🔍 Viewers
+
+;; Clerk comes with a number of useful built-in viewers e.g. for Clojure data, html & hiccup, tables, plots &c.
+
+;; When showing large data structures, Clerk's default viewers will show an excerpt of the data as to not overload the browser.
```

Comment:
this isn awkward usage of the word "exerpt". I suggest "pagination" as it's a familiar for developers. 
```suggestion
;; When showing large data structures, Clerk's default viewers will paginate the results."
```

could also be
```suggestion
;; When showing large data structures, Clerk's default viewers will initially only load the first few values".
```